posthog/api/property_definition.py

```diff
@@ -273,6 +282,48 @@ def _join_on_event_property(self):
             else ""
         )
 
+    @staticmethod
+    def __property_name_aliases():
+        """
+        Generates a SQL CASE statement that aliases property names to their label.
+        """
+        return (
+            "CASE name "
+            + " ".join(f"WHEN '{k}' THEN '{v}'" for (k, v) in PROPERTY_NAME_ALIASES.items())
+            + " ELSE name END as __name_alias"
+        )
+
+
+# See frontend/src/lib/taxonomy.tsx for where this came from
+PROPERTY_NAME_ALIASES = {
```

Comment:
How are these selected out of all the `KEY_MAPPING.event` items. Is there a way to automate this so that we don't forget to sync them?